fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -142,19 +142,21 @@ def build_upload_command(username, password, source = "/tmp")
         "-f '#{source}'",
         ENV["DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS"], # that's here, because the user might overwrite the -t option
         "-t 'Signiant'",
-        "-k 100000"
-      ].join(' ')
+        "-k 100000",
+        ("-itc_provider #{provider_short_name}" unless provider_short_name.to_s.empty?)
```

Comment:
Yes, it evaluates to `nil` unless the condition is met. Then the possible `nil` is removed by the `compact` at the end of the statement.